### PR TITLE
EPMRPP-45431 || System attributes retrieving

### PR DESCRIPTION
**Pull request by Yumfriez**
MERGE AFTER https://github.com/reportportal/agent-java-soapui/pull/14

### CODE REVIEW

File: readyapi/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 dependencies {
     compile project(':soapui')
-    compileOnly 'com.smartbear:ready-api-soapui-pro:2.3.0'
+    compileOnly 'com.smartbear.soapui:soapui:5.4.0'
```
Comment (HardNorth) [inline readyapi/build.gradle]: The same as for: https://github.com/reportportal/agent-java-soapui/pull/14/files#diff-78e2e48110a5f79a8d5e308fa7a8cf23R3

File: soapui/src/main/resources/agent.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+agent.name=agent-java-soapui
+agent.version=4.0.0-SNAPSHOT
```
Comment (HardNorth) [inline soapui/src/main/resources/agent.properties]: Please update version number

File: soapui/build.gradle [hunk 3]
```diff
@@ -2,12 +2,22 @@ dependencies {
...
     dependencies {
         exclude(dependency('com.smartbear.soapui:soapui'))
     }
-}
\ No newline at end of file
+    archiveName = "agent-java-${archiveName}"
+}
+
+task writeAgentProperties(type: WriteProperties) {
+    outputFile = file("${project.projectDir}/src/main/resources/agent.properties")
```
Comment (HardNorth) [inline soapui/build.gradle]: As per @ihar-kahadouski :https://github.com/reportportal/agent-java-jbehave/pull/35#discussion_r376355434

```
    {
    filesMatching('agent.properties') {
        expand(project.properties)
    }
```
and in properties file:

```
agent.name=${name}
agent.version=${version}
```

**Review by HardNorth — CHANGES_REQUESTED** [review]
Let's start with a painful thing. Unit tests?

File: soapui/build.gradle (hunk 3 above)
Comment (Yumfriez) [reply soapui/build.gradle]: updated

File: soapui/src/main/resources/agent.properties (hunk 2 above)
Comment (Yumfriez) [reply soapui/src/main/resources/agent.properties]: updated

File: readyapi/build.gradle (hunk 1 above)
Comment (Yumfriez) [reply readyapi/build.gradle]: 2.6.0 version for ready-api dependency set

File: soapui/build.gradle [hunk 4]
```diff
@@ -1,13 +1,24 @@
 dependencies {
-    compileOnly 'com.smartbear.soapui:soapui:5.4.0'
+    compile 'com.smartbear.soapui:soapui:5.4.0'
 
     compileOnly 'log4j:log4j:1.2.17'
-    compile 'com.epam.reportportal:client-java:4.0.4'
-    compile 'com.epam.reportportal:logger-java-log4j:4.0.1'
+    compile 'com.github.reportportal:client-java:591b7b41'
+    compile 'com.github.reportportal:commons-model:feec39f3'
+    compile 'com.epam.reportportal:logger-java-log4j:5.0.0-BETA-8'
```
Comment (HardNorth) [inline soapui/build.gradle]: BETA-9

File: soapui/build.gradle [hunk 5]
```diff
@@ -1,13 +1,24 @@
 dependencies {
-    compileOnly 'com.smartbear.soapui:soapui:5.4.0'
+    compile 'com.smartbear.soapui:soapui:5.4.0'
 
     compileOnly 'log4j:log4j:1.2.17'
-    compile 'com.epam.reportportal:client-java:4.0.4'
-    compile 'com.epam.reportportal:logger-java-log4j:4.0.1'
+    compile 'com.github.reportportal:client-java:591b7b41'
```
Comment (HardNorth) [inline soapui/build.gradle]: BETA-9

File: soapui/build.gradle [hunk 6]
```diff
@@ -1,13 +1,24 @@
...
+    compile 'com.github.reportportal:commons-model:feec39f3'
```
Comment (HardNorth) [inline soapui/build.gradle]: 5.0.0

**Review by HardNorth — CHANGES_REQUESTED** [review]
And please also update the branch with develop

File: soapui/build.gradle (hunk 4 above)
Comment (Yumfriez) [reply soapui/build.gradle]: updated

File: soapui/build.gradle (hunk 5 above)
Comment (Yumfriez) [reply soapui/build.gradle]: updated

File: soapui/build.gradle (hunk 6 above)
Comment (Yumfriez) [reply soapui/build.gradle]: updated

**Review by HardNorth — APPROVED** [review]

**Pull request merged**